### PR TITLE
TASK: Add documentation for CLI commands introduced with Neos 9

### LINKED ISSUE #4017: 9.0: Document new CLI commands
At least add some inline doc comments to make `./flow help <command>` work.
For example `./flow help cr:setup`

### PR DESCRIPTION
**Pull request by crydotsnake**
Adds the inline documentation for the new cli commands that where introduced with Neos 9.

Resolves: #4017

### CODE REVIEW

**Comment by crydotsnake** [comment]
## List of the Neos 9 CLI commands that needs documentation

### Package: Neos.ContentRepositoryRegistry

- [x] `./flow content:refreshrootnodedimensions`
- [x] `./flow content:movedimensionspacepoint`
- [x] `./flow content:createvariantsrecursively`
- [x] `./flow contentgraphintegrity:runviolationdetection`
- ~[ ] `./flow contentstream:export`~ (not implemented -> will be removed)
- [x] `./flow cr:prune`
- [x] `./flow structureadjustments:detect`
- [x] `./flow structureadjustments:fix`
- [x] `./flow subprocessprojectioncatchup:catchup`

### Package: Neos.ContentRepository.BehavioralTests

- [x] `./flow performancemeasurement:prepareperformancetest`
- [x] `./flow performancemeasurement:testperformance`

### Package: Neos.ContentRepository.BehavioralTests.ProjectionRaceConditionTester

- [x] `./flow raceconditiontracker:reset`
- [x] `./flow raceconditiontracker:analyzetrace`

**Comment by skurfuerst** [comment]
Thanks @crydotsnake  - let me know when this is ready from your side :)

**Comment by crydotsnake** [comment]
> Thanks @crydotsnake - let me know when this is ready from your side :)

Thanks @skurfuerst. I will try to continue working on it on the weekend.

But it would be nice, to get a introduction too the new introduced commands. Because i dont really know, what those commands are doing..

**Comment by bwaidelich** [comment]
@crydotsnake Thanks for starting this

> But it would be nice, to get a introduction too the new introduced commands.

Let me know when you want to talk about this one. For a couple of these I'll have to dig into the code myself. But I'm sure we can get some basic docs

**Comment by crydotsnake** [comment]
> @crydotsnake Thanks for starting this
> 
> > But it would be nice, to get a introduction too the new introduced commands.
> 
> Let me know when you want to talk about this one. For a couple of these I'll have to dig into the code myself. But I'm sure we can get some basic docs

That would be great. Maybe we can have a talk next week at thursday or friday?

**Comment by bwaidelich** [comment]
> That would be great. Maybe we can have a talk next week at thursday or friday?

@crydotsnake sure, just get in touch!

**Comment by bwaidelich** [comment]
@crydotsnake Maybe it's easier if you just pose all the questions you have here or in Slack so that we can tackle this one command at the time?

**Review by bwaidelich — APPROVED** [review]

**Review by dlubitz — APPROVED** [review]

File: Neos.ContentRepositoryRegistry/Classes/Command/SubprocessProjectionCatchUpCommandController.php [hunk 1]
```diff
@@ -21,6 +21,11 @@ class SubprocessProjectionCatchUpCommandController extends CommandController
      */
     protected $contentRepositoryRegistry;
 
+    /**
+     * @param string $contentRepositoryIdentifier
+     * @param string $projectionClassName
```
Comment (ahaeslich) [inline Neos.ContentRepositoryRegistry/Classes/Command/SubprocessProjectionCatchUpCommandController.php]: There are two new commands where you need to define the projection. If I remember correctly, then (currently) you have to specify the full class name in one. When using the other command, however, you can use an alias.

Which one is this? Or did it change lately and I have missed it?

Comment (bwaidelich) [inline Neos.ContentRepositoryRegistry/Classes/Command/SubprocessProjectionCatchUpCommandController.php]:
```suggestion
     * @param string $projectionClassName fully qualified class name of the projection to catch up
```

It's the FQN in this case (the command is internal though)

**Review by bwaidelich — APPROVED** [review]
Thanks!

**Pull request merged**